catho/quantum
# fix(package.json): Changed script to remove unit tests and snapshots folders from bundle

## Description
https://jirasoftware.catho.com.br/browse/QTM-446

## How to test
1. Clone the project
2. Checkout on branch CCOM-518
3. Run the yarn and yarn build commands (This command will create the "dist" folder with all the components) .
4. Check in each folder and subfolder of /dist that there is no file ending with unit.test.jsx or a folder called "snapshot". If so, request changes in the pull request stating the issue

Note: For testing purposes and to validate which files were deleted from the build, include the "-print" argument in the "clean:files:build" and "clean:directories:build" scripts. The commands should look something like `find ./dist -type f \\( -name '*.unit.test.js' -or -name '*.unit.test.jsx' \\) -print -delete` and `find ./dist -type d -name '__snapshots__' -print -exec rm -r {} +`, respectively

## Review guide
- [ ] Bundle
